src/components/Menu/index.tsx

```diff
@@ -144,8 +144,6 @@ const ClaimRewardButton = styled(ButtonPrimary)`
 export const NewLabel = styled.span`
   font-size: 10px;
   color: ${({ theme }) => theme.red};
-  height: calc(100% + 4px);
-  margin-left: 2px;
```

Comment:
useless, so removed

Reply:
is this affected?
<img width="201" alt="image" src="https://user-images.githubusercontent.com/11427213/196083783-6fb4ba03-d2a8-4c2b-8275-cb53cf12fc14.png">